src/index.js

```diff
@@ -3,6 +3,7 @@ import { version } from '../package.json';
 
 lazyload.version = version;
 lazyload.install = function (Vue) {
+    // eslint-disable-next-line no-undef
     Vue.directive(Slider.name, Slider);
```

Comment:
@VoIdJMp look at this part of the code. `Slider` is a global variable ?

Reply:
Yes bro its global

Reply:
Then we need to write some plugins for eslint to the reason, it should understand your globals.
 For example, if you have noticed this line of our code at eslint config:
```env: {
    browser: true,
    es6: true,
    'jest/globals': true,
  },```

Reply:
ok